src/aleph/sdk/client.py

```diff
@@ -608,22 +613,85 @@ async def get_posts(
             resp.raise_for_status()
             return await resp.json()
 
+    async def download_file_to_buffer(
+            self,
+            file_hash: str,
+            output_buffer: BinaryIO,
+            chunk_size: int,
+    ) -> None:
+        """
+        Download a file from the storage engine and write it to the specified output buffer.
+        :param self: The current instance of the class.
+        :param file_hash: The hash of the file to retrieve.
+        :param output_buffer: The binary output buffer to write the file data to.
+        :param chunk_size: Size of the chunk to download.
+        """
+        async with aiohttp.ClientSession() as session:
+            async with session.get(f"https://api1.aleph.im/api/v0/storage/raw/{file_hash}") as response:
+                response.raise_for_status()
+
+                while True:
+                    chunk = await response.content.read(chunk_size)
+                    if not chunk:
+                        break
+                    output_buffer.write(chunk)
+
+    async def download_file_ipfs_to_buffer(
+            self,
+            file_hash: str,
+            output_buffer: BinaryIO,
+            chunk_size : int,
+    ) -> None:
+        """
+        Download a file from the storage engine and write it to the specified output buffer.
+
+        :param file_hash: The hash of the file to retrieve.
+        :param output_buffer: The binary output buffer to write the file data to.
+        :param chunk_size: Size of chunk we download.
+        """
+        async with aiohttp.ClientSession() as session:
+            async with session.get(f"https://ipfs.io/ipfs/{file_hash}") as response:
```

Comment:
Can you use the IPFS server from aleph.im instead ?
```suggestion
            async with session.get(f"https://ipfs.aleph.im/ipfs/{file_hash}") as response:
```